src/xo-app/jobs/creation/index.js

```diff
@@ -0,0 +1,370 @@
+import ActionButton from 'action-button'
+import ActionRowButton from 'action-row-button'
+import find from 'lodash/find'
+import forEach from 'lodash/forEach'
+import GenericInput from 'json-schema-input/generic-input'
+import includes from 'lodash/includes'
+import isEmpty from 'lodash/isEmpty'
+import map from 'lodash/map'
+import mapValues from 'lodash/mapValues'
+import React, { Component } from 'react'
+import { error } from 'notification'
+import { SelectPlainObject } from '../helpers'
+import delay from 'lodash/delay'
+
+import {
+  apiMethods,
+  createJob,
+  deleteJob,
+  runJob,
+  subscribeJobs,
+  updateJob
+} from 'xo'
+
+const JOB_KEY = 'genericTask'
+
+const getType = function (param) {
+  if (!param) {
+    return
+  }
+  if (Array.isArray(param.type)) {
+    if (includes(param.type, 'integer')) {
+      return 'integer'
+    } else if (includes(param.type, 'number')) {
+      return 'number'
+    } else {
+      return 'string'
+    }
+  }
+  return param.type
+}
+
+/**
+ * Tries extracting Object targeted property
+ */
+const reduceObject = function (value, propertyName = 'id') {
```

Comment:
Use arrow functions when acceding `this` (or `arguments`) is not needed, especially when the only statement is a `return`.